modules/user.paths.yaml

```diff
@@ -1314,6 +1314,52 @@ user-image:
         description: Success
       default:
         description: Unexpected error.
+user-push:
+  post:
+    tags:
+      - User
+    summary: add push notification token to user
+    description: Add/update a device/platform specific token for push notifications
```

Comment:
Seems not very REST-like to use `POST` for updating. Should we have a dedicated `PUT` for that instead? Or is this acceptable?